src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -2510,20 +2564,69 @@ public void Inheritance_07()
     public abstract int X { get; }
     public virtual int Y { get; }
 }
-data class B(int X, int Y) : A
+abstract data class B1(int X, int Y) : A
+{
+}
+data class B2(int X, int Y) : A
 {
 }";
             var comp = CreateCompilation(source);
             comp.VerifyDiagnostics(
-                // (6,12): error CS0534: 'B' does not implement inherited abstract member 'A.X.get'
-                // data class B(int X, int Y) : A
-                Diagnostic(ErrorCode.ERR_UnimplementedAbstractMethod, "B").WithArguments("B", "A.X.get").WithLocation(6, 12));
+                // (9,12): error CS0534: 'B2' does not implement inherited abstract member 'A.X.get'
+                // data class B2(int X, int Y) : A
+                Diagnostic(ErrorCode.ERR_UnimplementedAbstractMethod, "B2").WithArguments("B2", "A.X.get").WithLocation(9, 12));
```

Comment:
>ERR_UnimplementedAbstractMethod [](start = 37, length = 31)

Error is incorrect per [https://github.com/dotnet/csharplang/pull/3527](https://github.com/dotnet/csharplang/pull/3527)

Reply:
Yes, we're not synthesizing overrides for `abstract` properties currently. That is tracked by https://github.com/dotnet/roslyn/issues/44785.